src/envattrs/__init__.py

```diff
@@ -1,17 +1,35 @@
 import os
 from typing import TypeVar, Type
 
-import attr
+try:
+    import attr
+except ImportError:
+    try:
+        import dataclasses
```

Comment:
```suggestion
try:
    import attr
except ImportError:
    attr = None
try:
    import dataclasses
except ImportError:
    dataclasses = None
```

Otherwise using envattrs with a dataclasses object won't work if attrs is installed.